src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,17 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value <= 0) {
+            return "0";
+        }
+
+        StringBuilder mirrorString = new StringBuilder();
+
+        while (value >= 1) {
+            mirrorString.append(value % 2);
+            value = value >> 1;
+        }
+
```

Comment:
you don't need to separate your code inside the method with empty lines, it's readable enough without it, and you make it longer

Reply:
Done